vcsclient/bitbucketcloud_test.go

```diff
@@ -243,6 +243,51 @@ func TestBitbucketCloud_GetLatestCommitInvalidPayload(t *testing.T) {
 	}
 }
 
+func TestBitbucketCloud_GetCloneInfo(t *testing.T) {
+	ctx := context.Background()
+
+	client, err := NewClientBuilder(vcsutils.BitbucketCloud).Build()
+	assert.NoError(t, err)
+
+	res, err := client.GetCloneInfo(ctx, owner, "jfrog-setup-cli")
```

Comment:
All the other tests use mocks, I'm not sure we should be making calls to bitbucket.org in this test.

Reply:
I have changed it to use mock data as well